gbpservice/tests/contrib/nfp_scripts/upgrade_nfp.sh

```diff
@@ -0,0 +1,146 @@
+#! /bin/bash
```

Comment:
Call this script as shift_mode.sh and keep it in gbpservice/nfp/scripts directory. Create this new directory.